tests/d3d12_device.c

```diff
@@ -336,11 +336,214 @@ static const DXGI_FORMAT depth_stencil_formats[] =
 void test_format_support(void)
 {
     D3D12_FEATURE_DATA_FORMAT_SUPPORT format_support;
+    D3D12_FEATURE_DATA_FEATURE_LEVELS feature_levels;
+    bool required_but_fails;
+    bool unspecified_format;
+    bool optional_format;
     ID3D12Device *device;
     ULONG refcount;
     unsigned int i;
+    unsigned int j;
+    unsigned int k;
     HRESULT hr;
 
+    static const DXGI_FORMAT all_formats[] =
+    {
+        DXGI_FORMAT_UNKNOWN,
```

Comment:
This looks like it's looping over all enums. Does this actually need to be an array?

Reply:
There are sections of the DXGI_FORMAT that are not defined. I can remove the array for range matching.